src/main/java/seedu/letsgethired/logic/commands/ViewCommand.java

```diff
@@ -0,0 +1,72 @@
+package seedu.letsgethired.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.letsgethired.commons.core.index.Index;
+import seedu.letsgethired.commons.util.ToStringBuilder;
+import seedu.letsgethired.logic.Messages;
+import seedu.letsgethired.logic.commands.exceptions.CommandException;
+import seedu.letsgethired.model.Model;
+import seedu.letsgethired.model.application.InternApplication;
+
+/**
+ * Selects and displays an Application identified using it's displayed index from the intern tracker.
+ */
+public class ViewCommand extends Command {
+    public static final String COMMAND_WORD = "view";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Selects and displays the intern application identified by the index number used in the "
+            + "displayed "
+            + "person list.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_VIEW_INTERN_APPLICATION_SUCCESS = "View application: %1$s";
```

Comment:
Perhaps we could change this to `Viewed intern application` to follow the convention of the other commands